### PR TITLE
add PropertyKind.vars()

### PR DESCRIPTION
**Pull request by okou19900722**
#163

### CODE REVIEW

**Comment by vietj** [comment]
can you add a test for this ?

File: src/test/java/io/vertx/test/codegen/testdataobject/PropertyKindDataObject.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import io.vertx.codegen.annotations.DataObject;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+@DataObject
+public interface PropertyKindDataObject {
+  int getInteger();
```
Comment (vietj) [inline src/test/java/io/vertx/test/codegen/testdataobject/PropertyKindDataObject.java]: why are the various getters needed on this interface ? I don't see how they are used in the test

Comment (okou19900722) [reply src/test/java/io/vertx/test/codegen/testdataobject/PropertyKindDataObject.java]: it's a kind of PropertyKind.VALUE

Comment (vietj) [reply src/test/java/io/vertx/test/codegen/testdataobject/PropertyKindDataObject.java]: I mean the getters like getInteger(), what are their use ?

Comment (okou19900722) [reply src/test/java/io/vertx/test/codegen/testdataobject/PropertyKindDataObject.java]: yes, it's another way to test, but i got a new way, now i had delete it

**Comment by okou19900722** [comment]
it's done

**Comment by okou19900722** [comment]
The master branch has changed a lot

**Comment by vietj** [comment]
Actually we are willing to discontinue the support of MVEL in the vertx stack (https://github.com/vert-x3/issues/issues/367) because it has issues with Java 9 and beyond, so this PR would have limited usage. 

Thanks for the contribution anyway, you can help on MVEL removal if you want.

**Comment by okou19900722** [comment]
OK, but I must first familiarize myself with the new code

**Comment by okou19900722** [comment]
hi,  I'm going to do something, but codegen lacks an interface to add a new GeneratorLoader.
Do you already have the code?

**Comment by vietj** [comment]
we need to add a META-INF/services mechanism for that

**Comment by vietj** [comment]
a contribution for it would be great :)

**Comment by vietj** [comment]
right now indeed it's hardcoded

**Comment by okou19900722** [comment]
#185 please take a look

**Pull request closed without merging**